<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDWaterWindow</class>
 <widget class="QMainWindow" name="WBSDWaterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Water system</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>951</width>
      <height>532</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item row="15" column="2">
      <widget class="QComboBox" name="cboUseDescaleCounters"/>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="cboUseWaterFilterCounters"/>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="lblDecalcCounter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Use descale counters(change it from false to true will  
 set the descale counter reset value to the factory value 150L)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="lblWaterFilter4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>water counter reset value  (A reset of the dispenser 
is needed to implement in the dispenser. A value of 0 
will set “ use waterfilter counters” to Disabled)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="lblWaterFilter3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Waterfilter warn level                   （Liters）  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="boilerTempU">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Boiler temperature (UPPER)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Boiler temperature (MID)</string>
       </property>
      </widget>
     </item>
     <item row="26" column="2">
      <widget class="QPushButton" name="btnResetDecalc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Start descaling routine</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="topTemp">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">gridline-color: rgb(0, 0, 0);</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Use waterfilter counters (change it from Disabled to Enabled will  
 set the Water counter reset value to the factory value 2000L)</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QPushButton" name="btnResetWaterfilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Reset waterfilter counter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Boiler temperature set-point (°C)</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="lblDescaleFilter3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>descale filter warn level               （Liters）  </string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="boilerTempM">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Heating</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Normal boiler level</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="lblWaterFilter2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Block machine on Waterfilter counter = 0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="levelState">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="lblWaterFilter1">
       <property name="text">
        <string>Waterfilter amount left counter     （Liters）</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="lblWaterFilterCounter">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="lblDescaleFilter4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>descale counter reset value(A value of 0  will set “ use descale counters” to Disabled)</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="blockTemp">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Block temperature set-point (°C)</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="lblDescaleFilter1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>descale amount left counter          （Liters）  </string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QComboBox" name="cboBlockOnDecalc">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Disable</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="heatingState">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="2">
      <widget class="QComboBox" name="cboBlockOnFilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Disable</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="lblDescaleFilter2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>block machine on descale=0</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QDoubleSpinBox" name="spiFilterResetVal">
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QDoubleSpinBox" name="spiDecalcResetVal">
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QDoubleSpinBox" name="spiDecalcWarn"/>
     </item>
     <item row="10" column="2">
      <widget class="QDoubleSpinBox" name="spiFilterWarn"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>topTemp</tabstop>
  <tabstop>blockTemp</tabstop>
  <tabstop>cboUseWaterFilterCounters</tabstop>
  <tabstop>cboBlockOnFilter</tabstop>
  <tabstop>btnResetWaterfilter</tabstop>
  <tabstop>cboUseDescaleCounters</tabstop>
  <tabstop>cboBlockOnDecalc</tabstop>
  <tabstop>btnResetDecalc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
